.travis.yml

```diff
@@ -9,6 +9,11 @@ env:
   - distribution: debian-testing
   - distribution: fedora
   - distribution: arch
+  - distribution: ubuntu-scan-build
```

Comment:
You can probably to the scan-builds in the above distributions, no need for running compilation both with and without scan build I think. Also exit with error if the html directory gets created, or maybe there is a scan-build option to report error if issues are found?